app/models/audit_results.rb

```diff
@@ -113,31 +100,22 @@ def remove_db_updated_results
     result_array.delete_if { |res_hash| DB_UPDATED_CODES.include?(res_hash.keys.first) }
   end
 
-  def report_results
-    # Report completed
-    report_completed(completed_results)
-
-    # Report errors
-    report_errors(error_results)
-    result_array
+  def results
+    result_array.dup.freeze
   end
 
   def completed_results
-    result_array.select(&result_ok?)
+    result_array.select(&result_ok?).freeze
   end
 
   def error_results
-    result_array.reject(&result_ok?)
+    result_array.reject(&result_ok?).freeze
   end
 
   def contains_result_code?(code)
     result_array.detect { |result_hash| result_hash.key?(code) } != nil
   end
 
-  def status_changed_to_ok?(result)
-    /to ok$/.match(result[AuditResults::CM_STATUS_CHANGED]) != nil
-  end
```

Comment:
Did this method move somewhere or is it just nuked? Asking because I think it's still invoked elsewhere in the `AuditResults` model.

Reply:
Ooops. You're right.